app/serializers/site_serializer.rb

```diff
@@ -288,23 +288,15 @@ def include_denied_emojis?
   end
 
   def tos_url
-    if SiteSetting.tos_url.present?
-      SiteSetting.tos_url
-    elsif SiteSetting.tos_topic_id > 0 && Topic.exists?(id: SiteSetting.tos_topic_id)
-      "#{Discourse.base_path}/tos"
-    end
+    Discourse.tos_path
   end
 
   def include_tos_url?
     tos_url.present?
   end
 
   def privacy_policy_url
```

Comment:
Should the attribute name be `privacy_policy_path` instead considering that we're calling `Discourse.privacy_policy_path`?

Reply:
I think the more appropriate name is `privacy_policy_url` because it can either be an absolute URL if the site setting is set or a relative path if the topic exists. I updated all occurrences of this.